DESCRIPTION

```diff
@@ -24,7 +24,7 @@ License: GPL (>= 2) | file LICENSE
 LazyLoad: yes
 Depends: R (>= 3.6.0)
 LinkingTo: Rcpp
-Imports: Matrix (>= 1.1-0), Rcpp (>= 0.11.0), stats, utils
-Suggests: inline, tinytest, pkgKitten, microbenchmark
+Imports: Rcpp (>= 0.11.0), stats, utils
+Suggests: Matrix, inline, microbenchmark, pkgKitten, tinytest
```

Comment:
Just a side comment and not a request to change:  When I modify code (particular "other people's code" I try to be minimal.  Adding `Matrix` was needed on the `Suggests:`, doing it at the front is fine, reordering the four remaining words just puts cognitive load here that we do not need.   (Again, no need to change, just a comment.)